analytics/transform.go

```diff
@@ -205,8 +218,65 @@ func (event ParsedEvent) GetValue(field string) (interface{}, error) {
 		}
 		return output, nil
 	}
+
 	return kvPairs[0].Value, nil
+}
+
+// GetUnstructEventValue returns the value for a provided atomic field inside an event's unstruct_event field
+func (event ParsedEvent) GetUnstructEventValue(path ...interface{}) (interface{}, error) {
+	fullPath := append([]interface{}{`data`, `data`}, path...)
+
+	el := json.Get([]byte(event[indexMap["unstruct_event"]]), fullPath...)
+	return el.GetInterface(), el.LastError()
+}
+
+// GetContextValue returns the value for a provided atomic field inside an event's contexts or derived_contexts
+func (event ParsedEvent) GetContextValue(contextName string, path ...interface{}) (interface{}, error) {
```

Comment:
For this release this will get the job done. 

We are still parsing a JSON string to a map and then back to a JSON string, so I've raised an issue to return to it and explore whether we can improve efficiency: https://github.com/snowplow/snowplow-golang-analytics-sdk/issues/23